fortitude/src/lib.rs

```diff
@@ -95,7 +96,14 @@ pub trait AstRule {
     /// Apply a rule over some text, generating all violations raised as a result.
     fn apply(source: &SourceFile) -> anyhow::Result<Vec<Diagnostic>> {
```

Comment:
Oh, we should be able to drop this function entirely now?

Reply:
It's still used in the test for `DeprecatedRelationalOperator`, but otherwise yes.

Reply:
Just saw that you already raised a PR into this branch to fix it, so that's sorted now